k8s/crd-samples/deployment-cstor-pool.yaml

```diff
@@ -29,10 +29,14 @@ spec:
           mountPath: /dev     
         - name: tmp
           mountPath: /tmp
-          mountPropagation: Bidirectional
+#         mountPropagation: Bidirectional
+        lifecycle:
+          postStart:
+            exec:
+              command: ["/bin/sh", "-c", "sleep 15"]
```

Comment:
Why do we start both the containers with "-c" "sleep 15" cmd-line arguments?

Reply:
Since this is WIP and we have decided to move them to openebs/elves, I'll address the review comments in a separate PR. @kmova @avishnu